x/tokenfactory/types/codec.go

```diff
@@ -23,10 +26,25 @@ func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 		&MsgCreateDenom{},
 		&MsgMint{},
 		&MsgBurn{},
-		// &MsgForceTransfer{},
+		// &MsgForceTransfer{}, // TODO: why commented?
```

Comment:
oh, this is my comment. I would delete the comment, since it's the same comment in osmosis: https://github.com/osmosis-labs/osmosis/blob/ccc07c3d1831e84dbbf35fb842ff3973f08e912f/x/tokenfactory/types/codec.go#L28